libs/barista-components/filter-field/src/filter-field.ts

```diff
@@ -339,6 +342,9 @@ export class DtFilterField<T = any>
     delay(0),
   );
 
+  /** Whether the filter-field is being interacted with */
+  interactionState: boolean;
+
```

Comment:
You are never setting the interactionState. 
It might be a good idea to create a method that will do a couple of things: 
1. set the interactionState member
2. Checks if the interactionState has changed in regards to the previous value
3. If 2., emit the interactionStateChange.